storage-shared-benchmarking/src/main/java/com/google/cloud/storage/benchmarking/StorageSharedBenchmarkingCli.java

```diff
@@ -108,35 +120,68 @@ private void runWorkload1() {
     StorageOptions retryStorageOptions =
         StorageOptions.newBuilder().setProjectId(project).setRetrySettings(retrySettings).build();
     Storage storageClient = retryStorageOptions.getService();
-    runW1R3(storageClient);
+    try {
+      ListeningExecutorService executorService =
+          MoreExecutors.listeningDecorator(Executors.newFixedThreadPool(workers));
+      for (int i = 0; i < samples; i++) {
+        if (warmup > 0) {
+          runWarmup(storageClient);
+        }
+        runW1R3(storageClient, false, executorService);
+      }
+    } catch (Exception e) {
+      System.err.println("Failed to run workload 1: " + e.getMessage());
+    }
   }
 
   private void runWorkload4() {
     RetrySettings retrySettings = StorageOptions.getDefaultRetrySettings().toBuilder().build();
     StorageOptions retryStorageOptions =
         StorageOptions.grpc().setRetrySettings(retrySettings).setAttemptDirectPath(true).build();
     Storage storageClient = retryStorageOptions.getService();
-    runW1R3(storageClient);
+    try {
+      ListeningExecutorService executorService =
+          MoreExecutors.listeningDecorator(Executors.newFixedThreadPool(workers));
+      for (int i = 0; i < samples; i++) {
+        if (warmup > 0) {
+          runWarmup(storageClient);
+        }
+        runW1R3(storageClient, false, executorService);
+      }
+    } catch (Exception e) {
+      System.err.println("Failed to run workload 4: " + e.getMessage());
```

Comment:
IIUC, you'll want to output this error as a `FAIL` status instead which means making status an option in: https://github.com/googleapis/java-storage/blob/4c7ab68396892766f11f0ae22e10887cb6878e95/storage-shared-benchmarking/src/main/java/com/google/cloud/storage/benchmarking/W1R3.java#L111

Reply:
I added cloudmonitoring results for failures in the callable but left this in just for anything else (mainly so the IDE is happy)